build.gradle

```diff
@@ -286,10 +287,11 @@ subprojects {
     }
   }
 
-  // Exclude tests which are known to be flaky in the Travis CI environment
+  // Exclude tests which are known to be flaky in the Travis CI environment. Since all test tasks are TestNG tasks
+  // EXCEPT the integTest task, exclude the options from being evaluated by travis for the integTest task.
   if (System.getenv('TRAVIS') == 'true' && System.getenv('USER') == 'travis') {
     afterEvaluate {
-      project.tasks.withType(Test).forEach {
+      project.tasks.withType(Test).matching { it.name != "integTest" }.forEach {
```

Comment:
I suggest:
```
project.tasks.withType(Test) {
  if (it.options instanceof TestNGOptions) {
    it.options.excludeGroups 'ci-flaky'
    it.systemProperties['test.httpRequestTimeout'] = '20000'
  }
}
```

Reply:
good idea, it's better than what I was doing.